test/integration/smart_answer_flows/business_coronavirus_support_finder_test.rb

```diff
@@ -39,4 +39,29 @@ class BusinessCoronavirusSupportFinderFlowTest < ActiveSupport::TestCase
       end
     end
   end
+
```

Comment:
Somewhat related: could we merge the contexts in the existing test in this file, so the two tests are structured consistently?